playwright-e2e/tests/rgp/adult-enrollment.spec.ts

```diff
@@ -9,10 +9,17 @@ import TellUsAboutYourFamilyPage from 'pages/rgp/enrollment/tell-us-about-your-f
 import TellUsYourStoryPage, { WHO } from 'pages/rgp/enrollment/tell-us-your-story-page';
 import HomePage from 'pages/rgp/home-page';
 import { setAuth0UserEmailVerified } from 'utils/api-utils';
+import { setPatientParticipantGuid } from 'utils/faker-utils';
+import dsmHome from 'pages/dsm/home-page';
+import * as dsmAuth from 'authentication/auth-dsm';
+import Select from 'lib/widget/select';
+import { Navigation } from 'lib/component/dsm/navigation/navigation';
+import ParticipantListPage from 'pages/dsm/participantList-page';
+import { StudyNav } from 'lib/component/dsm/navigation/enums/studyNav.enum';
 
-const { RGP_USER_EMAIL, RGP_USER_PASSWORD } = process.env;
+const { RGP_USER_EMAIL, RGP_USER_PASSWORD, DSM_USER_EMAIL, DSM_USER_PASSWORD } = process.env;
 
-test.describe('Adult Self Enrollment', () => {
+test.describe.serial('Adult Self Enrollment', () => {
```

Comment:
Will take a look at the test and fix it post haste

Reply:
kibana seems to be slow - so it might be more than a couple of mins

Reply:
correction - I used test env when checking local run - that's why it's working for me - new RGP pts since about March 10th don't seem to be appearing in DSM and that's why the test failed - looking into it

Reply:
*don't seem to be appearing in DSM dev I mean